Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Horde Invasion

## Changelog:
- **The Horde Invasion:**
   - The Horde is scripted to conquer Azeroth and Khaz Modan. Its Warchief and vassals get special troops. The Warchief gets the Great Invader modifier giving high dread.
   - The Horde CB gives 50% of conquered duchies to the strongest clans. The rest goes to the Warchief.
   - The Horde falls apart if it takes enough lands for the orcs or the year is 613.
- **Title name changes:**
   - Orcish feudal and clan counts are called **_Overlords_**.
   - Orcish feudal and clan dukes are called **_High Overlords_**.
   - Rulers of the Horde are called **_Warchiefs_**.
   - Orcish kings are called **_Warlords_**.
   - The Horde is no longer **_High Kingdom of Horde_**.
- The orcish clans and the Horde start with the **_Orcish Elective_** law: a marshal, knights and powerful vassals can vote and can be elected.
- The Warchief can use the **_Invite Remaining Clans_** decision summoning the Burning Blade and Dragonmaw clans and getting more special troops.

## Developer changelog:
- Added `purge_historical_character_effect`.

# How to test:
Observe or play the Horde in the 583 start date and tell your opinion.
[Click Here](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/20/files?file-filters%5B%5D=.yml) to proofread localization files.
